firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/common/SessionReportingCoordinator.java

```diff
@@ -34,7 +34,7 @@
  * This class handles Crashlytics lifecycle events and manages capture, persistence, and sending of
```

Comment:
Should we use any language of "SessionReportingCoordinator"? e.g., "This Class coordinates Crashlytics lifecycle events and manages the capture, persistence, and sending of sessions as reports to Firebase Crashlytics"?

Reply:
Good point.

Reply:
Done.